ecosystem/sep-0008.md

```diff
@@ -226,6 +226,8 @@ Name | Type | Description
 
 This response means that the user must complete an action before this transaction can be approved. The approval service will provide a URL that facilitates the action. Upon completion, the user will resubmit the transaction.
 
+The action the user must complete is interactive, and is intended to be displayed in a browser. The client may be able to skip the need to display the URL in the browser if it can supply the requested details.
```

Comment:
How well-known is the concept of an "interactive" flow? I had never heard of this concept before Stellar so maybe we could include a description about what it means.

Reply:
Great point. The term `interactive` is unnecessary and raises more questions than the text has without it. Rather than define the word I removed its use in 1421ab9.